packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -62,12 +62,13 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
       scripts: scriptsSourceMap,
       vendor: vendorSourceMap,
     },
-    optimization: {
-      styles: stylesOptimization,
-      scripts: scriptsOptimization,
-    },
   } = buildOptions;
 
+  const {
+    styles: stylesOptimization,
+    scripts: scriptsOptimization,
+  } = normalizeOptimization(buildOptions.optimization);
```

Comment:
At this stage the `buildOptions.optimization` are already normalized.

https://github.com/angular/angular-cli/blob/97aa1fb7cc7595ca696fbe72fdf4982dd9815dfd/packages/angular_devkit/build_angular/src/utils/normalize-builder-schema.ts#L51

Reply:
That works for angular internals, but what about 3rd party libs using public api?

https://github.com/storybookjs/storybook/blob/bf61334046da41dbf1580d95a3faae5305a0deb7/app/angular/src/server/angular-cli_utils.ts#L86

Reply:
These are all private APIs which shouldn’t be used by 3rd parties.

Anything which is imported using deep imports or not from `@angular-devkit/build-angular` is private.

Reply:
@alan-agius4 normalizeBrowserSchema is also private -- if I were to work on a PR for @storybook/angular, what would be the public api I should look for?  Is there a document you can point me to?

Reply:
Unfortunately there isn’t a documentation for the public API.

Although all the public API methods can be found here: https://github.com/angular/angular-cli/blob/cfb368408b532e488acaa1a5a921240f0dc7dfad/packages/angular_devkit/build_angular/src/index.ts#L30

Reply:
Thanks for feedback, is there a reason you're opposed to this fix?

Reply:
The reason being is that its redundant in the supported scenarios.